--- rpmdrake-5.46/rpmdrake.orig	2013-11-07 19:18:09.829594260 +0000
+++ rpmdrake-5.46/rpmdrake	2013-11-07 19:18:46.650596197 +0000
@@ -639,11 +639,11 @@
          $license =~ s/\n/\n\n/sg; # nicer formatting
          my $w = gtknew('AboutDialog', name => N("Rpmdrake"),
                         version => $Rpmdrake::init::version,
-                        copyright => N("Copyright (C) %s by Mandriva", '2002-2009'),
+                        copyright => N("Copyright (C) %s by OpenMandriva", '2002-2013'),
                         license => $license, wrap_license => 1,
-                        comments => N("Rpmdrake is Mandriva Linux package management tool."),
-                        website => 'http://mandrivalinux.com',
-                        website_label => N("Mandriva Linux"),
+                        comments => N("Rpmdrake is OpenMandriva Lx package management tool."),
+                        website => 'http://www.openmandriva.org',
+                        website_label => N("OpenMandriva"),
                         authors => 'Thierry Vignaud <vignaud@mandriva.com>',
                         artists => 'Hélène Durosini <ln@mandriva.com>',
                         translator_credits =>
--- rpmdrake-5.46/Rpmdrake/edit_urpm_sources.pm.orig	2013-11-07 19:19:57.805599939 +0000
+++ rpmdrake-5.46/Rpmdrake/edit_urpm_sources.pm	2013-11-07 19:20:50.734602724 +0000
@@ -977,11 +977,11 @@
                $license =~ s/\n/\n\n/sg; # nicer formatting
                my $w = gtknew('AboutDialog', name => N("Rpmdrake"),
                               version => $rpmdrake::distro_version,
-                              copyright => N("Copyright (C) %s by Mandriva", '2002-2008'),
+                              copyright => N("Copyright (C) %s by OpenMandriva", '2002-2013'),
                               license => $license, wrap_license => 1,
-                              comments => N("Rpmdrake is Mandriva Linux package management tool."),
-                              website => 'http://mandrivalinux.com',
-                              website_label => N("Mandriva Linux"),
+                              comments => N("Rpmdrake is OpenMandriva Lx package management tool."),
+                              website => 'http://www.openmandriva.org',
+                              website_label => N("OpenMandriva"),
                               authors => 'Thierry Vignaud <vignaud@mandriva.com>',
                               artists => 'Hélène Durosini <ln@mandriva.com>',
                               translator_credits =>
