commit 5ea0b8c2fde94f669cd6fadc3b295107fe2b06c6
Author: Crispin Boylan <crisb@mandriva.org>
Date:   Mon Apr 14 20:50:48 2014 +0100

    Fix asking for locale related packages (bug #662)

diff --git a/Rpmdrake/gui.pm b/Rpmdrake/gui.pm
index 5b30e2d..8bc8ece 100644
--- a/Rpmdrake/gui.pm
+++ b/Rpmdrake/gui.pm
@@ -697,12 +697,6 @@ sub is_locale_available {
 sub callback_choices {
     my (undef, undef, undef, $choices) = @_;
     return $choices->[0] if $::rpmdrake_options{auto};
-    foreach my $pkg (@$choices) {
-        foreach ($pkg->requires_nosense) {
-            /locales-/ or next;
-            is_locale_available($_) and return $pkg;
-        }
-    }
     my $callback = sub { interactive_msg(N("More information on package..."), get_info($_[0]), scroll => 1) };
     $choices = [ sort { $a->name cmp $b->name } @$choices ];
     my @choices = interactive_list_(N("Please choose"), (scalar(@$choices) == 1 ? 
