--- rpmdrake-5.26.12/Rpmdrake/gui.pm.orig	2013-01-17 13:11:52.341847695 +0400
+++ rpmdrake-5.26.12/Rpmdrake/gui.pm	2013-01-17 13:12:28.549032670 +0400
@@ -1081,6 +1081,8 @@
 
 sub run_help_callback {
     my (undef, $url) = @_;
+    my ($user) = grep { $_->[2] eq $ENV{USERHELPER_UID} } list_passwd();
+    local $ENV{HOME} = $user->[7] if $user && $ENV{USERHELPER_UID};
     run_program::raw({ detach => 1, as_user => 1 }, 'www-browser', $url);
 }
 
