--- rpmdrake-5.46/rpmdrake.pm~	2013-04-15 23:36:15.000000000 +0100
+++ rpmdrake-5.46/rpmdrake.pm	2013-09-20 09:47:25.667059704 +0100
@@ -632,7 +632,7 @@
 Please check that your network is currently running.
 
 Is it ok to continue?")
-: N("I need to contact the Mandriva website to get the mirror list.
+: N("I need to contact the OpenMandriva website to get the mirror list.
 Please check that your network is currently running.
 
 Is it ok to continue?");
--- rpmdrake-5.46/Rpmdrake/edit_urpm_sources.pm~	2013-04-15 23:36:15.000000000 +0100
+++ rpmdrake-5.46/Rpmdrake/edit_urpm_sources.pm	2013-09-18 22:36:12.529395245 +0100
@@ -135,7 +135,7 @@
 N("This will attempt to install all official sources corresponding to your
 distribution (%s).
 
-I need to contact the Mandriva website to get the mirror list.
+I need to contact the OpenMandriva website to get the mirror list.
 Please check that your network is currently running.
 
 Is it ok to continue?", $distro),
--- rpmdrake-5.46/Rpmdrake/pkg.pm~	2013-04-15 23:36:15.000000000 +0100
+++ rpmdrake-5.46/Rpmdrake/pkg.pm	2013-09-18 22:35:51.217394382 +0100
@@ -273,7 +273,7 @@
 		my ($mirror) = choose_mirror($urpm, transient => $w->{real_window} || $::main_window,
                                        message => join("\n\n",
                                                        N("You have no configured update media. MandrivaUpdate cannot operate without any update media."),
-                                                       N("I need to contact the Mandriva website to get the mirror list.
+                                                       N("I need to contact the OpenMandriva website to get the mirror list.
 Please check that your network is currently running.
 
 Is it ok to continue?"),
