diff -Nur rpmdrake-5.26.12old/Rpmdrake/open_db.pm rpmdrake-5.26.12/Rpmdrake/open_db.pm
--- rpmdrake-5.26.12old/Rpmdrake/open_db.pm	2012-02-24 17:30:13.000000000 +0400
+++ rpmdrake-5.26.12/Rpmdrake/open_db.pm	2013-08-23 17:33:10.230771885 +0400
@@ -141,7 +141,7 @@
     if (is_it_a_devel_distro()) {
         grep { !$_->{ignore} } @{$urpm->{media}};
     } else {
-        grep { !$_->{ignore} && $_->{update} } @{$urpm->{media}};
+        grep { !$_->{ignore} } @{$urpm->{media}};
     }
 }
 
