diff -aur rpmdrake-5.46.old/Rpmdrake/pkg.pm rpmdrake-5.46/Rpmdrake/pkg.pm
--- rpmdrake-5.46.old/Rpmdrake/pkg.pm	2015-03-17 01:49:40.090984808 -0400
+++ rpmdrake-5.46/Rpmdrake/pkg.pm	2015-03-17 01:50:20.535218336 -0400
@@ -34,7 +34,7 @@
 use Rpmdrake::gurpm;
 use Rpmdrake::formatting;
 use Rpmdrake::rpmnew;
-use gurpm::RPMProgressDialog;
+use gurpm::Gtk2RPMProgressDialog;
 
 use rpmdrake;
 use urpm;
@@ -734,7 +734,7 @@
     # select packages to uninstall for !update mode:
     perform_removal($urpm, { map { $_ => $pkgs->{$_} } @to_remove }) if !$probe_only_for_updates;
 
-    $gurpm = gurpm::RPMProgressDialog->new($urpm);
+    $gurpm = gurpm::Gtk2RPMProgressDialog->new($urpm);
     $gurpm->label(1 ? N("Please wait") : N("Package installation..."));
     my $_gurpm_clean_guard = before_leaving { $gurpm->destroy };
     my $something_installed;
@@ -774,9 +774,9 @@
                                  undef $lock;
                                  undef $rpm_lock;
                              },
-                             inst => \&gurpm::RPMProgressDialog::callback_inst,
-                             trans => \&gurpm::RPMProgressDialog::callback_inst,
-                             uninst => \&gurpm::RPMProgressDialog::callback_inst,
+                             inst => \&gurpm::Gtk2RPMProgressDialog::callback_inst,
+                             trans => \&gurpm::Gtk2RPMProgressDialog::callback_inst,
+                             uninst => \&gurpm::Gtk2RPMProgressDialog::callback_inst,
                              ask_yes_or_no => sub {
                                  # handle 'allow-force' and 'allow-nodeps' options:
                                  my ($title, $msg) = @_;
@@ -789,7 +789,7 @@
                                  interactive_msg($title, $message, scroll => 1);
                              },
                              # cancel installation when 'cancel' button is pressed:
-                             trans_log => sub { gurpm::RPMProgressDialog::callback_download(@_) }, # or goto return_with_exit_code },
+                             trans_log => sub { gurpm::Gtk2RPMProgressDialog::callback_download(@_) }, # or goto return_with_exit_code },
                              post_extract => sub {
                                  my ($_set, $transaction_sources, $transaction_sources_install) = @_;
                                  $done += grep { !/\.src\.rpm$/ } values %$transaction_sources;         #updates
@@ -814,7 +814,7 @@
                                      yesno => 1, text => { no => N("Cancel"), yes => N("Ok") },
                                  );
                              },
-                             pre_check_sig => \&gurpm::RPMProgressDialog::callback_pre_check_sig,
+                             pre_check_sig => \&gurpm::Gtk2RPMProgressDialog::callback_pre_check_sig,
                              #check_sig => sub { $gurpm->set_progressbar(++$progress/$total) },
                              bad_signature => sub {
                                  my ($msg, $msg2) = @_;
@@ -916,7 +916,7 @@
     }
 
   return_with_exit_code:
-    $something_installed = gurpm::RPMProgressDialog::get_something_done();
+    $something_installed = gurpm::Gtk2RPMProgressDialog::get_something_done();
     return !($something_installed || scalar(@to_remove));
 }
 
@@ -927,7 +927,7 @@
     my ($urpm, $pkgs) = @_;
     my @toremove = map { if_($pkgs->{$_}{selected}, $pkgs->{$_}{urpm_name}) } keys %$pkgs;
     return if !@toremove;
-    my $gurpm = gurpm::RPMProgressDialog->new($urpm);
+    my $gurpm = gurpm::Gtk2RPMProgressDialog->new($urpm);
     $gurpm->label(1 ? N("Please wait") : N("Please wait, removing packages..."));
     $gurpm->init_progressbar;
     my $_gurpm_clean_guard = before_leaving { $gurpm->destroy };
@@ -945,7 +945,7 @@
 	    @results = $::rpmdrake_options{parallel}
 		? urpm::parallel::remove($urpm, \@toremove)
 		: urpm::install::install($urpm, \@toremove, {}, {},
-                                   callback_uninst => \&gurpm::RPMProgressDialog::callback_inst,
+                                   callback_uninst => \&gurpm::Gtk2RPMProgressDialog::callback_inst,
                                );
 	    open_rpm_db('force_sync');
 	},
