--- rpmdrake-5.46/Rpmdrake/gui.pm.orig	2013-10-28 09:31:15.985088778 +0000
+++ rpmdrake-5.46/Rpmdrake/gui.pm	2013-10-28 09:31:27.776089398 +0000
@@ -207,7 +207,7 @@
     my ($pkg, $upkg, $installed_version, $raw_medium) = @_;
     my $a = ugtk2::markup_to_TextView_format(
         $spacing . join("\n$spacing",
-                        format_field(N("Version: ")) . $upkg->EVR,
+                        eval { format_field(N("Version: ")) . $upkg->EVR },
                         ($upkg->flag_installed ?
                            format_field(N("Currently installed version: ")) . $installed_version : ()
                         ),
