diff -Naur rpmdrake-5.26.12.orig/Rpmdrake/pkg.pm rpmdrake-5.26.12/Rpmdrake/pkg.pm
--- rpmdrake-5.26.12.orig/Rpmdrake/pkg.pm	2013-01-22 11:16:04.523239364 +0400
+++ rpmdrake-5.26.12/Rpmdrake/pkg.pm	2013-01-22 11:18:21.463722660 +0400
@@ -638,7 +638,7 @@
     $w->set_sensitive(0);
     my $_restore_sensitive = before_leaving { $w->set_sensitive(1) };
 
-    my $_flush_guard = Gtk2::GUI_Update_Guard->new;
+    #my $_flush_guard = Gtk2::GUI_Update_Guard->new;
 
     if (my $group = get_parallel_group()) {
         return perform_parallel_install($urpm, $group, $w, \$statusbar_msg_id);
